test/exercises/duplicates/forStudents/README

```diff
@@ -0,0 +1,172 @@
+Data Processing Hands-On Exercise: Finding Duplicate Files
+==========================================================
+
+This directory contains the framework for writing a Chapel program to find
+duplicate files.
+
+What is in this directory?
+**************************
+
+FileHashing.chpl:
+  This module defines:
+  * a type SHA256Hash supporting the usual comparison and assignment operators
+    as well as writeln
+  * a function computeFileHash that returns a path
+  * a function relativeRealPath used to normalize paths
+
+duplicates.chpl:
+  This is the code you'll edit for this exercise.
+
+example-files/
+  This directory stores some example files for testing your program.
+
+expected-output:
+  This file contains the expected output for your program when run on the
+  example-files directory.
+
+Hands-On Tasks
+**************
+
+Task 1: Getting Started
+-----------------------
+
+To get started with this project, compile 'duplicates.chpl'.  Note that this
+program uses the OpenSSL library to compute SHA256 hashes and might need
```

Comment:
We may want to include example compilation command here for users trying this on their local machines. Also, we could suggest settings `LDFLAGS` and `CPPFLAGS` to avoid throwing the linker flags every time.

Reply:
This no longer matters :)